### ISSUE TITLE
Users can't login after 1.1.0 update

### ISSUE THREAD

**Issue by Censtudios** [humhub/humhub#1783]
#1770 I think you misunderstood. This issue is not happening because of the API. I was just able to bypass it thanks to the API.

If not for making my own login form that uses the API, users would still not be able to login after the 1.1.0 update. The issue must be related to the changes made in the 1.1.0 auth system.

**Comment by luke-** [comment]
Maybe you themed the login form and not migrated it?

https://www.humhub.org/docs/guide-theming-migrate.html#migrate-to-11

**Comment by Censtudios** [comment]
No, the same issue exists when I use the same theme that's in the 1.1.0 fresh install

_(On a side note I see that you mention that in 1.2 you want to split the .css file into more files. Isn't that worse for the loading time of the site? Having users download more files will only slow down the loading speed)_

**Comment by buddha87** [comment]
Just the less files are split. And the less files will be compiled to one css file.

**Comment by luke-** [comment]
@Censtudios That bug really weird, are you using any customziations (e.g. themes, modules)?

**Comment by luke-** [comment]
Are you using LDAP?

**Comment by Censtudios** [comment]
No I'm not using LDAP. Just the normal authentication. The facebook authentication works, but then users must've already had it connected to their account (which they couldn't, because before 1.1.0 it wasn't available). Normal "password" authentication does not work.

**Comment by luke-** [comment]
Can you test your database against a clean humhub installation without any changes?

**Comment by Censtudios** [comment]
Will do. I will need some time to set that up

**Comment by cgx** [comment]
This is the PHP error I get when trying to authenticate once migrated to v1.1.0:

    PHP Fatal error:  Call to a member function getIsEnabled() on a non-object in /var/www/humhub.git/protected/humhub/modules/dashboard/components/actions/DashboardStream.php on line 30, referer: https://domain.tld/humhub/index.php

**Comment by vickysw** [comment]
I have fecing this issue. anyone can help me?
